api/src/main/java/com/datatorrent/api/Context.java

```diff
@@ -161,6 +161,13 @@
      */
     Attribute<Class<?>> TUPLE_CLASS = new Attribute<>(new Class2String<>());
 
+    /**
+     * Attribute of input port.
+     * This is a read-only attribute to query whether the input port is connected to a DelayOperator
+     * This is for iterative processing.
+     */
+    Attribute<Boolean> IS_CONNECTED_TO_DELAY_OPERATOR = new Attribute<>(false);
```

Comment:
Should this be even an attribute? This set internally by platform so why expose it as an attribute..